kafka-producer/src/main/resources/config/base.conf

```diff
@@ -1,13 +1,32 @@
 {
   "haystack": {
-    "directory": "extractors/"
+    "kafka": {
+      "brokers": "localhost",
+      "port": 9092,
+      "fromtopic": "proto-spans",
+      "totopic": "",
+      "threadcount": 1,
+      "sessiontimeout": 15000,
+      "maxwakeups": 10,
+      "wakeuptimeoutms": 3000,
+      "polltimeoutms": 250,
+      "commitms": 3000
+    },
+    "externalKafkaList": [{
+      "brokers": "localhost:9092",
+      "port": 9093,
+      "totopic": "externalKafkaTopic",
+      "acks": "0",
+      "batchsize": 8192,
+      "lingerms": 4,
+      "buffermemory": 1024,
+      "extractorList": ["JsonExtractor"] // Default Extractor
```

Comment:
Are we going to support this? Piping multiple extractors while writing to the kafka.